cmd/system-probe/modules/ping.go

```diff
@@ -0,0 +1,138 @@
+//go:build linux
+
+package modules
+
+import (
+	"encoding/json"
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/DataDog/datadog-agent/cmd/system-probe/api/module"
+	"github.com/DataDog/datadog-agent/cmd/system-probe/config"
+	sysconfigtypes "github.com/DataDog/datadog-agent/cmd/system-probe/config/types"
+	"github.com/DataDog/datadog-agent/cmd/system-probe/utils"
+	pingcheck "github.com/DataDog/datadog-agent/pkg/networkdevice/pinger"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+	"github.com/gorilla/mux"
+	"go.uber.org/atomic"
+	"google.golang.org/grpc"
+)
+
+const (
+	countParam    = "count"
+	intervalParam = "interval"
+	timeoutParam  = "timeout"
+)
+
+type pinger struct{}
+
+// Pinger is a factory for NDMs Ping module
+var Pinger = module.Factory{
+	Name:             config.PingModule,
+	ConfigNamespaces: []string{},
+	Fn: func(cfg *sysconfigtypes.Config) (module.Module, error) {
+		return &pinger{}, nil
+	},
+	NeedsEBPF: func() bool {
+		return false
+	},
+}
+
+var _ module.Module = &pinger{}
+
+func (p *pinger) GetStats() map[string]interface{} {
+	return nil
+}
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏No need to implement this?

Reply:
It is part of the system-probe `Module` interface